.travis.yml

```diff
@@ -3,10 +3,12 @@ language: c
 compiler:
   - i686-w64-mingw32-gcc
   - x86_64-w64-mingw32-gcc
+  - gcc
 
 addons:
   apt:
     packages:
+      - gcc
```

Comment:
hint: 64bit compiler is in package `gcc` and 32bit compiler is in package `gcc-i686-linux-gnu`